.mypy.ini

```diff
@@ -0,0 +1,58 @@
+# Global options:
```

Comment:
@kamathhrishi would you mind if we kept configuration in setup.cfg rather than .mypi.ini?  I believe that is [also idiomatic](https://mypy.readthedocs.io/en/stable/config_file.html)).  The reason I like setup.cfg is that it allows us to keep configurations for multiple tools in a single file, which in the past has streamlined code review when we make multiple synchronized config changes in a single PR.

Reply:
I moved it to setup.cfg.